samples/outlook/35-notifications/add-getall-remove.yaml

```diff
@@ -127,34 +148,51 @@ template:
         </section>
 
         <section class="samples ms-font-m">
-          <h3>Try it out</h3>
+          <h2>Try it out</h3>
```

Comment:
From my view, it appears we have an <h2> tag being closed with an </h3> tag. Was the intent to close it with an </h2> tag instead?

Reply:
```suggestion
          <h3>Try it out</h3>
```

Reply:
OR...
```suggestion
          <h2>Try it out</h2>
```